rpc_state_reader/src/sir_state_reader.rs

```diff
@@ -270,14 +271,15 @@ pub fn execute_tx_configurable_with_state(
                     Address(Felt252::from_bytes_be_slice(
                         tx.sender_address().0.key().bytes(),
                     )),
-                    match tx {
+                    // TODO[0.13] Properly convert between V3 tx fields
```

Comment:
@fmoletta Could you create an issue for this, and link it in the comment?

Reply:
Should be part of #1199